website/common/script/libs/shops.js

```diff
@@ -224,10 +225,40 @@ shops.getSeasonalShopCategories = function getSeasonalShopCategories (user, lang
     fall2015Warrior: i18n.t('scarecrowWarriorSet', language),
   };
 
+  let availableSpells = [
```

Comment:
Can do! I was following the pattern established by the existing `availableSets`, but that works too.